operators/test/e2e/volume_test.go

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package e2e
+
+import (
+	"testing"
+
+	"github.com/elastic/cloud-on-k8s/operators/test/e2e/helpers"
+	"github.com/elastic/cloud-on-k8s/operators/test/e2e/stack"
+)
+
+func TestVolumeDefaultPVC(t *testing.T) {
+	k := helpers.NewK8sClientOrFatal()
+
+	initStack := stack.NewStackBuilder("test-es-default-pvc").
+		WithESMasterNodes(1, stack.DefaultResources)
+
+	helpers.TestStepList{}.
+		WithSteps(stack.InitTestSteps(initStack, k)...).
+		WithSteps(stack.CreationTestSteps(initStack, k)...).
+		WithSteps(stack.ESClusterChecks(initStack.Elasticsearch, k)...).
+		WithSteps(stack.CheckDefaultPVC(initStack.Elasticsearch, k)).
```

Comment:
should we include this call in `CheckStackSteps()`, to be performed for every test suite (default case), and add a single new test for empty dirs instead of adding 2 new tests?

Reply:
Sgtm, will add another commit

Reply:
@sebgl if you could re-LGTM and check if I understood your idea correctly ....